CHANGELOG.md

```diff
@@ -6,6 +6,42 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project does not yet adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 for setuptools_scm/PEP 440 reasons.
 
+## 1.2.3 Chia blockchain 2021-07-21
+
+### Added
+
+- Added ability to change payout instructions in the GUI.
+- Added an option to revert to sequential read. There are some systems (primarily macos+exfat) where the parallel read features results in very long lookup times. This addition makes the parallel feature the default, but adds the ability to disable it and revert back to sequential reads.
+- Added backwards compatibility for Coin Solutions in push_tx since renaming it to CoinSpend.
+- Added an option to set the default constants on the simulator.
+- Added a warning to user to not send money to the pool contract address.
+- Added new CLI command: 'chia plotnft show' to show plot count
```

Comment:
This command already existed - the output was expanded to include the plot count. It was also changed to output more information about pool points.

Reply:
Oh, and this last one is thanks to @mishan

Reply:
This command already existed. I just added an output line that displays the % of successful points over the last 24 hours

Reply:
@mishan added the % succesful points, we should capture that in the changelog @paulhainsworth-chia